seeds/dex/trades/dex_trades_seed.csv

```diff
@@ -146,6 +146,15 @@ bnb,biswap,1,2022-11-22,0x9050a1e2f7aa392e39195b5b22382d595e94104b72699655b198c4
 bnb,biswap,1,2022-11-22,0xe2efd633720bd29322da7e39857e98b3c9904a1c55f9b3995441160983e06296,119,0xe9e7cea3dedca5984780bafc599bd69add087d56,43.898605549510836795,0x563ca064e41f3b5d80adeecfe49ab375fd7afbef,2756.0281
 bnb,biswap,1,2022-11-22,0x0b9ba3ee4a0b3f4db2baba70be527f5a50aea77423cd921106ec9f0ac8dcf169,115,0xbb4cdb9cbd36b01bd1cbaebf2de08d9173bc095c,0.018248512111984361,0xe9e7cea3dedca5984780bafc599bd69add087d56,4.837104443141354752 
 bnb,biswap,1,2022-11-22,0xaf03946d9b98e5d6e841bb50555208801e48d773a8dfdf1ecc203d943c27c603,350,0x2170ed0880ac9a755fd29b2688956bd959f933f8,6.247796965743473757,0x7130d2a12b9bcbfae4f2634d864a1ee1ce3ead9c,0.43849
+bnb,sushiswap,1,2022-11-22,0x683b4299466b4fec3fac2859e2dd27215f69879b31a7384593b46aef71952ed5,373,0xa0ed3c520dc0632657ad2eaaf19e26c4fd431a84,6571.818499746365907562,0xe9e7cea3dedca5984780bafc599bd69add087d56,20
+bnb,sushiswap,1,2022-11-22,0xa572f3ea4bcc491c3d6a3b54f2cd59c776238c3187c1ee7975d9635f8a55d129,260,0xe9e7cea3dedca5984780bafc599bd69add087d56,30.330984668706979348,0xa0ed3c520dc0632657ad2eaaf19e26c4fd431a84,10000
+bnb,sushiswap,1,2022-11-22,0xfac7881f1461c9c56a11f68647385ea27c4d66849f646d5f377e558014652df2,536,0xe9e7cea3dedca5984780bafc599bd69add087d56,0.003061259047161221,0xa0ed3c520dc0632657ad2eaaf19e26c4fd431a84,1
+bnb,sushiswap,1,2023-01-17,0x5169181339080e2d56c13c0a075dac19f19e3f8300a8c068e8f4519bf424e7ab,186,0xe9e7cea3dedca5984780bafc599bd69add087d56,2.487085655648097925,0xc598275452fa319d75ee5f176fd3b8384925b425,177.678
+polygon,sushiswap,1,2022-11-22,0xcd4fda9b8492b26fb927a37f2276066d4e845d5831898c6aa3bc52195e64be29,51,0x4c3bf0a3de9524af68327d1d2558a3b70d17d42a,10,0x2791bca1f2de4661ed88a30c99a7a9449aa84174,17.471735
+polygon,sushiswap,1,2022-11-22,0xf993683656e9d580099b359bf697a15d2c704bac55a5341df59d266e00bd2214,95,0x2b9e7ccdf0f4e5b24757c1e1a80e311e34cb10c7,10,0x2791bca1f2de4661ed88a30c99a7a9449aa84174,23.507209
+polygon,sushiswap,1,2022-11-22,0x581a3ed5905f52623813259451c9bba0fb4a1b902bd2c84a04bf26831384221c,33,0xd125443f38a69d776177c2b9c041f462936f8218,160.621170255560769709,0x7ceb23fd6bc0add59e62ac25578270cff1b9f619,0.041137
+polygon,sushiswap,1,2022-11-22,0xb3fd8d3ea108e9bb55221d12e9c3d8a315e5eb1c7aad884817cdfe1d10ac0d35,38,0xd125443f38a69d776177c2b9c041f462936f8218,61.238323253823349251,0x960d43be128585ca45365cd74a7773b9d814dfbe,33.502071
+polygon,sushiswap,1,2022-11-22,0x65c15eccd09b70a984d39c61acd4d3e7b5dfa461ac761ccfdc87bae92077391f,42,0xd125443f38a69d776177c2b9c041f462936f8218,26.343507111310067208,0x60ed6acef3a96f8cdaf0c0d207bbafa66e751af2,0.087984
```

Comment:
<table role="table" class="table_table__fuS_N" style="margin: 0px; padding: 0px; box-sizing: inherit; width: 1382.43px; border-collapse: collapse; font-family: &quot;IBM Plex Mono&quot;; font-size: 1.2rem;"><thead style="margin: 0px; padding: 0px; box-sizing: inherit;"><tr role="row" style="margin: 0px; padding: 0px; box-sizing: inherit;"><th colspan="1" role="columnheader" aria-sort="none" style="margin: 0px; padding: 0.5rem 1rem; box-sizing: inherit; text-align: inherit; font-weight: 500; position: sticky; top: 0px; height: 3rem; white-space: nowrap; background: var(--color1-200);"><button type="button" title="Toggle SortBy" style="margin: 0px; padding: 0px; box-sizing: inherit; font-weight: inherit; cursor: pointer; text-align: inherit; font-size: inherit; font-family: inherit; line-height: inherit; background: transparent; border: none; color: inherit; display: block; width: 151.203px; user-select: text;"><div class="" style="margin: 0px; padding: 0px; box-sizing: inherit;">test_description</div></button></th><th colspan="1" role="columnheader" aria-sort="none" style="margin: 0px; padding: 0.5rem 1rem; box-sizing: inherit; text-align: inherit; font-weight: 500; position: sticky; top: 0px; height: 3rem; white-space: nowrap; background: var(--color1-200);"><button type="button" title="Toggle SortBy" style="margin: 0px; padding: 0px; box-sizing: inherit; font-weight: inherit; cursor: pointer; text-align: inherit; font-size: inherit; font-family: inherit; line-height: inherit; background: transparent; border: none; color: inherit; display: block; width: 100.805px; user-select: text;"><div class="" style="margin: 0px; padding: 0px; box-sizing: inherit;">result (model)</div></button></th><th colspan="1" role="columnheader" aria-sort="none" style="margin: 0px; padding: 0.5rem 1rem; box-sizing: inherit; text-align: inherit; font-weight: 500; position: sticky; top: 0px; height: 3rem; white-space: nowrap; background: var(--color1-200);"><button type="button" title="Toggle SortBy" style="margin: 0px; padding: 0px; box-sizing: inherit; font-weight: inherit; cursor: pointer; text-align: inherit; font-size: inherit; font-family: inherit; line-height: inherit; background: transparent; border: none; color: inherit; display: block; width: 108px; user-select: text;"><div class="" style="margin: 0px; padding: 0px; box-sizing: inherit;">expected (seed)</div></button></th><th colspan="1" role="columnheader" aria-sort="none" style="margin: 0px; padding: 0.5rem 1rem; box-sizing: inherit; text-align: inherit; font-weight: 500; position: sticky; top: 0px; height: 3rem; white-space: nowrap; background: var(--color1-200);"><button type="button" title="Toggle SortBy" style="margin: 0px; padding: 0px; box-sizing: inherit; font-weight: inherit; cursor: pointer; text-align: inherit; font-size: inherit; font-family: inherit; line-height: inherit; background: transparent; border: none; color: inherit; display: block; width: 115.203px; user-select: text;"><div class="visual_datetime__8Dkmt" style="margin: 0px; padding: 0px; box-sizing: inherit; white-space: nowrap;">block_date</div></button></th><th colspan="1" role="columnheader" aria-sort="none" style="margin: 0px; padding: 0.5rem 1rem; box-sizing: inherit; text-align: inherit; font-weight: 500; position: sticky; top: 0px; height: 3rem; white-space: nowrap; background: var(--color1-200);"><button type="button" title="Toggle SortBy" style="margin: 0px; padding: 0px; box-sizing: inherit; font-weight: inherit; cursor: pointer; text-align: inherit; font-size: inherit; font-family: inherit; line-height: inherit; background: transparent; border: none; color: inherit; display: block; width: 72px; user-select: text;"><div class="" style="margin: 0px; padding: 0px; box-sizing: inherit;">blockchain</div></button></th><th colspan="1" role="columnheader" aria-sort="none" style="margin: 0px; padding: 0.5rem 1rem; box-sizing: inherit; text-align: inherit; font-weight: 500; position: sticky; top: 0px; height: 3rem; white-space: nowrap; background: var(--color1-200);"><button type="button" title="Toggle SortBy" style="margin: 0px; padding: 0px; box-sizing: inherit; font-weight: inherit; cursor: pointer; text-align: inherit; font-size: inherit; font-family: inherit; line-height: inherit; background: transparent; border: none; color: inherit; display: block; width: 64.8047px; user-select: text;"><div class="" style="margin: 0px; padding: 0px; box-sizing: inherit;">project</div></button></th><th colspan="1" role="columnheader" aria-sort="none" style="margin: 0px; padding: 0.5rem 1rem; box-sizing: inherit; text-align: inherit; font-weight: 500; position: sticky; top: 0px; height: 3rem; white-space: nowrap; background: var(--color1-200);"><button type="button" title="Toggle SortBy" style="margin: 0px; padding: 0px; box-sizing: inherit; font-weight: inherit; cursor: pointer; text-align: inherit; font-size: inherit; font-family: inherit; line-height: inherit; background: transparent; border: none; color: inherit; display: block; width: 50.4062px; user-select: text;"><div class="" style="margin: 0px; padding: 0px; box-sizing: inherit;">version</div></button></th><th colspan="1" role="columnheader" aria-sort="none" style="margin: 0px; padding: 0.5rem 1rem; box-sizing: inherit; text-align: inherit; font-weight: 500; position: sticky; top: 0px; height: 3rem; white-space: nowrap; background: var(--color1-200);"><button type="button" title="Toggle SortBy" style="margin: 0px; padding: 0px; box-sizing: inherit; font-weight: inherit; cursor: pointer; text-align: inherit; font-size: inherit; font-family: inherit; line-height: inherit; background: transparent; border: none; color: inherit; display: block; width: 475.203px; user-select: text;"><div class="" style="margin: 0px; padding: 0px; box-sizing: inherit;">tx_hash</div></button></th><th colspan="1" role="columnheader" aria-sort="none" style="margin: 0px; padding: 0.5rem 1rem; box-sizing: inherit; text-align: inherit; font-weight: 500; position: sticky; top: 0px; height: 3rem; white-space: nowrap; background: var(--color1-200);"><button type="button" title="Toggle SortBy" style="margin: 0px; padding: 0px; box-sizing: inherit; font-weight: inherit; cursor: pointer; text-align: inherit; font-size: inherit; font-family: inherit; line-height: inherit; background: transparent; border: none; color: inherit; display: block; width: 64.8047px; user-select: text;"><div class="" style="margin: 0px; padding: 0px; box-sizing: inherit;">evt_index</div></button></th></tr></thead><tbody role="rowgroup" style="margin: 0px; padding: 0px; box-sizing: inherit;"><tr role="row" style="margin: 0px; padding: 0px; box-sizing: inherit;"><td role="cell" style="margin: 0px; padding: 0.5rem 1rem; box-sizing: inherit; white-space: nowrap; border-bottom: 1px solid var(--color1-200); color: rgb(0, 0, 0); font-family: &quot;IBM Plex Mono&quot;; font-size: 12px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="" style="margin: 0px; padding: 0px; box-sizing: inherit;"><div style="margin: 0px; padding: 0px; box-sizing: inherit;">matched records count</div></div></td><td role="cell" style="margin: 0px; padding: 0.5rem 1rem; box-sizing: inherit; white-space: nowrap; border-bottom: 1px solid var(--color1-200); color: rgb(0, 0, 0); font-family: &quot;IBM Plex Mono&quot;; font-size: 12px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="" style="margin: 0px; padding: 0px; box-sizing: inherit;">0</div></td><td role="cell" style="margin: 0px; padding: 0.5rem 1rem; box-sizing: inherit; white-space: nowrap; border-bottom: 1px solid var(--color1-200); color: rgb(0, 0, 0); font-family: &quot;IBM Plex Mono&quot;; font-size: 12px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="" style="margin: 0px; padding: 0px; box-sizing: inherit;">1</div></td><td role="cell" style="margin: 0px; padding: 0.5rem 1rem; box-sizing: inherit; white-space: nowrap; border-bottom: 1px solid var(--color1-200); color: rgb(0, 0, 0); font-family: &quot;IBM Plex Mono&quot;; font-size: 12px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="visual_datetime__8Dkmt" style="margin: 0px; padding: 0px; box-sizing: inherit; white-space: nowrap;">2022-11-22 00:00</div></td><td role="cell" style="margin: 0px; padding: 0.5rem 1rem; box-sizing: inherit; white-space: nowrap; border-bottom: 1px solid var(--color1-200); color: rgb(0, 0, 0); font-family: &quot;IBM Plex Mono&quot;; font-size: 12px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="" style="margin: 0px; padding: 0px; box-sizing: inherit;"><div style="margin: 0px; padding: 0px; box-sizing: inherit;">polygon</div></div></td><td role="cell" style="margin: 0px; padding: 0.5rem 1rem; box-sizing: inherit; white-space: nowrap; border-bottom: 1px solid var(--color1-200); color: rgb(0, 0, 0); font-family: &quot;IBM Plex Mono&quot;; font-size: 12px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="" style="margin: 0px; padding: 0px; box-sizing: inherit;"><div style="margin: 0px; padding: 0px; box-sizing: inherit;">sushiswap</div></div></td><td role="cell" style="margin: 0px; padding: 0.5rem 1rem; box-sizing: inherit; white-space: nowrap; border-bottom: 1px solid var(--color1-200); color: rgb(0, 0, 0); font-family: &quot;IBM Plex Mono&quot;; font-size: 12px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="" style="margin: 0px; padding: 0px; box-sizing: inherit;">1</div></td><td role="cell" style="margin: 0px; padding: 0.5rem 1rem; box-sizing: inherit; white-space: nowrap; border-bottom: 1px solid var(--color1-200); color: rgb(0, 0, 0); font-family: &quot;IBM Plex Mono&quot;; font-size: 12px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="" style="margin: 0px; padding: 0px; box-sizing: inherit;"><div style="margin: 0px; padding: 0px; box-sizing: inherit;">0x65c15eccd09b70a984d39c61acd4d3e7b5dfa461ac761ccfdc87bae92077391f</div></div></td><td role="cell" style="margin: 0px; padding: 0.5rem 1rem; box-sizing: inherit; white-space: nowrap; border-bottom: 1px solid var(--color1-200); color: rgb(0, 0, 0); font-family: &quot;IBM Plex Mono&quot;; font-size: 12px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><br class="Apple-interchange-newline">

Reply:
https://dune.com/queries/1908357
This is an unmatched seed.
Polygonscan: https://polygonscan.com/tx/0x65c15eccd09b70a984d39c61acd4d3e7b5dfa461ac761ccfdc87bae92077391f

(It seems this record is not shown in the model query result, but it should be a sushiswap polygon trade record)